src/app/static/src/js/components/runWorkflow/runWorkflowCreate.vue

```diff
@@ -17,9 +17,9 @@
                           v-model="selectedWorkflow"
                           placeholder="Search by name or user...">
                     <template id="optionTemplate" #option="{ name, email, date }">
-                        <div>{{ name }}
-                            <span style="opacity: 0.5; float:right;">
-                            {{ email }} | {{ getLongTimestamp(date) }}</span>
+                        <div class="overflow-hidden">{{ name }}
```

Comment:
This isn't ideal because then the content is hidden if too long/on smaller screens.

Ditch `overflow-hidden` and `float:right`.

Then I reckon either add

```
.vs__dropdown-option--highlight {
  display: inline-block;
}
```

so that the highlight covers the whole item (this opion looks better on smaller screens but slightly worse on larger ones.) Or fix the color here so that the text doesn't disappear when the highlight applies - i.e. use the `text-muted` class instead of applying the `opacity 0.5` style

Reply:
adding text-muted works fairly well on all display but text alignment  is not good.

Reply:
Sorry yeah you'll have to change the `span` to a `div` as well (if we want it on 2 lines?) Or if we want it on one line copy the template used in `reportList.vue`: https://github.com/vimc/orderly-web/blob/master/src/app/static/src/js/components/runReport/reportList.vue#L5